src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -159,6 +161,10 @@ private async ValueTask InitializeAsync(PackageManagerModel model, INuGetUILogge
                 controller.PackageManagerControl = this;
             }
 
+            ISourceRepositoryProvider sourceRepositoryProvider = await ServiceLocator.GetComponentModelServiceAsync<ISourceRepositoryProvider>();
+            var sourceRepositories = sourceRepositoryProvider.GetRepositories();
+            _packageVulnerabilityService = new PackageVulnerabilityService(sourceRepositories, _uiLogger);
```

Comment:
Sources can change. You probably want to listen to settings changes and reload the sources.

Reply:
Made note of this for a follow-up. Thanks!